index.html

```diff
@@ -1,66 +1,282 @@
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Morbi venenatis at odio vitae tincidunt. Etiam sollicitudin sit amet leo quis molestie. <span class="higlight-3">
+      Morbi eu mauris viverra, bibendum odio id, finibus erat. </span>  Sed rhoncus lacinia porttitor. Duis eleifend libero erat,
+      sit amet semper augue elementum quis. Proin leo sem, ultricies vitae augue pulvinar, dignissim tempus tellus.
+      Proin molestie nulla tortor, et auctor dolor pretium vitae. Donec posuere dui vel ante porttitor aliquet.
+      Ut dictum gravida turpis a lacinia. Pellentesque nunc lorem, condimentum a consequat at, luctus eu ante.
+      Cras nec dignissim lacus, at luctus odio. Nunc a lorem pharetra, vulputate libero auctor, venenatis dolor.
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Morbi venenatis at odio vitae tincidunt. Etiam sollicitudin sit amet leo quis molestie. <span class="higlight-3">
+      Morbi eu mauris viverra, bibendum odio id, finibus erat. </span>  Sed rhoncus lacinia porttitor. Duis eleifend libero erat,
+      sit amet semper augue elementum quis. Proin leo sem, ultricies vitae augue pulvinar, dignissim tempus tellus.
+      Proin molestie nulla tortor, et auctor dolor pretium vitae. Donec posuere dui vel ante porttitor aliquet.
+      Ut dictum gravida turpis a lacinia. Pellentesque nunc lorem, condimentum a consequat at, luctus eu ante.
+      Cras nec dignissim lacus, at luctus odio. Nunc a lorem pharetra, vulputate libero auctor, venenatis dolor.
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Morbi venenatis at odio vitae tincidunt. Etiam sollicitudin sit amet leo quis molestie. <span class="higlight-3">
+      Morbi eu mauris viverra, bibendum odio id, finibus erat. </span>  Sed rhoncus lacinia porttitor. Duis eleifend libero erat,
+      sit amet semper augue elementum quis. Proin leo sem, ultricies vitae augue pulvinar, dignissim tempus tellus.
+      Proin molestie nulla tortor, et auctor dolor pretium vitae. Donec posuere dui vel ante porttitor aliquet.
+      Ut dictum gravida turpis a lacinia. Pellentesque nunc lorem, condimentum a consequat at, luctus eu ante.
+      Cras nec dignissim lacus, at luctus odio. Nunc a lorem pharetra, vulputate libero auctor, venenatis dolor.
+      Donec dui nulla, ultricies et convallis at, maximus non ante. Sed pellentesque vel arcu vel venenatis.
+      Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+    </p>
+
+    <div class="flex-box">
+      <div>
+        <img class="text-image" src="https://picsum.photos/300/300" alt="text-image" loading="lazy">
```

Comment:
Az `alt` paraméterről elfelejtettem elmondani, hogy tetszőleges rövid szöveg írható bele, akár egy pársoros is, ha segít a látási problémával küzdőknek "látni" a kép tartalmát. Tudom, ez itt most placeholder kép, nem tudod mi lesz itt, így nehéz lenne ennél pontosabbat írni. Ezesetben egyébként simán elég egy "Ez a kép csak egy illusztráció a szöveg jobb elválasztása érdekében".